rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -808,7 +808,7 @@ void gs_frame::take_screenshot(std::vector<u8> data, const u32 sshot_width, cons
 
 void gs_frame::mouseDoubleClickEvent(QMouseEvent* ev)
 {
-	if (m_disable_mouse || g_cfg.io.move == move_handler::mouse) return;
+	if (m_disable_mouse || g_cfg.io.move == move_handler::mouse || g_cfg.io.move == move_handler::gun) return;
```

Comment:
This change makes no sense unless your gun handler triggers mouse click events for some reason

Reply:
if you do not put this line, double gun pressure switch to fullscreen while this looks seems to take any device.

Reply:
This is weird. Does your gun fire mouse events? It may be some configuration issue on your end.